tests/cephadm/test_cephadm_ansible_wrapper.py

```diff
@@ -0,0 +1,110 @@
+import os
+
+from cli.cephadm.ansible import Ansible
+from cli.cephadm.cephadm import CephAdm
+from cli.cephadm.exceptions import (
+    CephadmAnsiblePlaybookExecutionError,
+    CephadmOpsExecutionError,
+)
+from cli.utilities.packages import Package, Rpm
+from cli.utilities.utils import get_node_ip
+from utility.install_prereq import (
+    ConfigureCephadmAnsibleInventory,
+    ConfigureCephadmRepositories,
+    EnableToolsRepositories,
+)
+from utility.log import Log
+
+log = Log(__name__)
+
+CEPHADM_ANSIBLE_PATH = "/usr/share/cephadm-ansible"
+CEPHADM_PREFLIGHT_PLAYBOOK = "cephadm-preflight.yml"
+CEPHADM_PREFLIGHT_VARS = {"ceph_origin": "rhcs"}
+
+
+class ConfigNotFoundError(Exception):
+    pass
+
+
+def put_cephadm_ansible_playbook(node, playbook):
+    """Put playbook to cephadm ansible location.
+    Args:
+        playbook (str): Playbook need to be copied to cephadm ansible path
+    """
+    dst = os.path.join(CEPHADM_ANSIBLE_PATH, os.path.basename(playbook))
+
+    node.upload_file(sudo=True, src=playbook, dst=dst)
+    log.info(f"Uploaded playbook '{playbook}' to '{dst}' on node.")
+
+
+def run(ceph_cluster, **kwargs):
+    """Module to execute cephadm-ansible wrapper playbooks
+
+    Example:
+        - test:
+            name: Bootstrap cluster using cephadm-ansible playbook
+            desc: Execute playbooks/bootstrap-cluster.yaml
+            config:
+                playbook: bootstrap-cluster.yaml
```

Comment:
`    args = kwargs.get("config", {}).get("args")`

please add example of the args.

Reply:
Updated Testsuite yaml file config with appropriate example